windows/snowagent.config

```diff
@@ -43,6 +43,7 @@
         <!-- only files with the following extensions are included in scan -->
         <IncludeCriteria>
             <FileType>exe</FileType>
+			<FileType>jar</FileType>
```

Comment:
indentation. also the jar scanner should pick those up without activation. corrent @PerMalmberg ?

Reply:
Yes, jar files are automatically added to the includes when jar scanning is enabled.

Reply:
removed the filetype for jar